test/unit/__mocks__/shared-config-merge.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  extends: 'my-lint-staged-config',
```

Comment:
Thanks for the PR, but I don't think this is a good way to achieve extensions. We purposefully removed support for "_advanced_" config and all its keys are supposed to be file globs.

The only way to do this without breaking existing behavior would be via cli options like `--extends my-lint-staged-config`, at least that I can think of.

Reply:
In that case...we don't actually need extends yet, so just going to use, e.g. `npx lint-staged -c node_modules/@org/shared-config/lint-staged`

Reply:
I'm pretty sure you can also do `-c @org/shared-configs/lint-staged` since `lilconfig` should be able to resolve from `node_modules/`. Let me know!

Reply:
Seems to work, thanks!